assets/js/apps/search/components/Result.jsx

```diff
@@ -102,23 +108,29 @@ function Result({ result }) {
               type="button"
               onClick={dataToggler}
               className="btn btn-link"
-              data-target="#data"
+              data-bs-target="#data"
               name="showChild"
             >
               <i className={dataToggleClass ? 'fa fa-plus' : 'fa fa-minus'} />
             </button>
             <a style={{ fontWeight: 'bold', display: 'inline-block' }} href={result.url}>{result.title}</a>
-            <ul className="nav nav-pills badgelist pull-right" style={{ display: 'inline-block' }}>
-              <li className="pull-right">
-                <span className="badge badge-info mr-2" title={`Date Created: ${result.created_time}`}>
+            <ul className="nav nav-pills badgelist float-end" style={{ display: 'inline-block' }}>
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933941)
```suggestion
            <ul className="nav nav-pills badgelist float-end" style={{ display: "inline-block" }}>
```